app/routers/agenda.py

```diff
@@ -39,6 +40,18 @@ def agenda(
     """Route for the agenda page, using dates range or exact amount of days."""
 
     user_id = 1  # there is no user session yet, so I use user id- 1.
+
+    start_date_graph, end_date_graph = calc_dates_range_for_agenda(0, 0, 7)
+
+    events_objects_for_graph = agenda_events.get_events_per_dates(
```

Comment:
The name of the variables here can be a bit better :)

Reply:
done